TACC/Core-Styles
# fix: s-footer link color affects buttons

## Overview / Changes

- Do **not** let `s-footer` link color affect buttons.
- Add style for an `<hr>` inside an `s-footer`.

## Related

- required by https://github.com/TACC/tup-ui/pull/476

## Testing

1. Open http://localhost:3000/components/detail/s-footer--complex.
2. Verify button has **white** text.
3. Hover over button.
4. Verify button **still** has white text.

## UI

https://github.com/user-attachments/assets/8f0ec4a5-2f49-4ed1-9fc7-a66f034f699a
